conf/application.conf

```diff
@@ -88,7 +88,8 @@ microservice {
   }
 }
 
-list-of-available-journeys = "SMP,STD,CAN,SUB,CON,SIM"
+list-of-available-journeys = "CRT,CAN,SUB,CON"
```

Comment:
Similar here, I'm not sure that we should have Submission as available journey, journey and declaration is the same. Journey and choice is a different thing. Choice can have values that doesn't exists in journey, but it should have just a Choice that will allow it. I think in internal app @MaciejRewera added handling this. So the Submissions is choice that contains just redirection

Reply:
Depends what we mean by "journey" and "choice".  I took "list-of-available-journeys" to mean "options that are available on the Choice page" - so that we can turn them on or off.

Similar with "list-of-available-declarations" - we just want the ability to hide them via config.

I don't think we need to get hung-up on what is a journey